pkgs/development/interpreters/bqn/cbqn/cbqn-bytecode.nix

```diff
@@ -5,13 +5,13 @@
 
 stdenvNoCC.mkDerivation {
   pname = "cbqn-bytecode";
-  version = "unstable-2023-01-27";
+  version = "unstable-2023-04-19";
 
```

Comment:
Split to another commit

Reply:
Why? cbqn and bytecode should be updated together, atomically.

Reply:
As @sternenseemann said, `cbqn-bytecode` should match the submodule used by `CBQN`. If done in two commits, the first commit will result in a version that either doesn't build correctly or doesn't work correctly.

Reply:
Also, how exactly this will mismatch? Is this a serious argument?
 
- Ofborg will build both "atomically"; there is no risk of mismatch

- When merged, the end users will see the ending result "atomically", not the intermediary steps; there is no risk of mismatch

- It is not uncommon to include multiple commits in a same PR. Indeed this is the typical practice when a program requires a NixOS module and a test suite. Example: kavita

- If the "atomic jumbo commit" made sense, then cimg, gmic and gmic-qt should be a jumbo commit, but they obviously aren't:

Reply:
The magic words are `git bisect` and `git cherry-pick`. 
A more granular git history is more helpful to track changes.